go-nvgpu/gpu_adapter.go

```diff
@@ -0,0 +1,99 @@
+package nvgpu
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/confidentsecurity/go-nvtrust/pkg/gonvtrust"
+	"github.com/intel/trustauthority-client/go-connector"
+)
+
+type GPUEvidence struct {
+	Evidence      string                  `json:"evidence"`
+	Certificate   string                  `json:"certificate"`
+	Nonce         string                  `json:"gpu_nonce"`
```

Comment:
Missing the field “Arch” to specify the architecture of NVGPU such as "HOPPER" for future supported archs. thanks.